xbmc/GUIPassword.h

```diff
@@ -60,6 +60,8 @@ class CGUIPassword : public ISettingCallback
    */
   bool CheckSettingLevelLock(const SettingLevel& level, bool enforce = false);
   bool CheckMenuLock(int iWindowID);
+  bool IsVideoUnlocked();
```

Comment:
Could this method be cost?

Reply:
No - due to  `IsMasterLockUnlocked(true)` being called

Reply:
This is all a big mess. My next question would be why IsMasterLockUnlocked is not const. But this goes probably beyond the scope of this PR.

Reply:
Agreed. Beyond scope.